app/main.py

```diff
@@ -1,2 +1,101 @@
+from typing import Any, Generator
+
+
 class Dictionary:
-    pass
+
+    def __init__(self) -> None:
+        self.max_capacity = 8
+        self.threshold = self.max_capacity - (self.max_capacity * 2 // 3)
```

Comment:
The calculation for the threshold might not be ideal. You're using self.max_capacity - (self.max_capacity * 2 // 3), which is equivalent to self.max_capacity // 3, meaning the table resizes when it's about 1/3rd full instead of 2/3rds full as typically expected with hash tables.

`
self.threshold = int(self.max_capacity * 2 / 3)`